website/docs/d/stream_instance.html.markdown

```diff
@@ -10,7 +10,7 @@ Describes a Stream Instance.
 
 `mongodbatlas_stream_instance` describes a stream instance.
 
--> **NOTE:** Atlas Streams is currently in Preview, you'll need to set the environment variable `MONGODB_ATLAS_ENABLE_BETA=true` to use this data source. To learn more about stream processing and existing limitations, see the [Atlas Stream Processing Documentation](https://www.mongodb.com/docs/atlas/atlas-sp/overview/#atlas-stream-processing-overview).
+-> **NOTE:** Atlas Streams is currently in preview, you'll need to set the environment variable `MONGODB_ATLAS_ENABLE_BETA=true` to use this data source. To learn more about stream processing and existing limitations, see the [Atlas Stream Processing Documentation](https://www.mongodb.com/docs/atlas/atlas-sp/overview/#atlas-stream-processing-overview).
```

Comment:
When you view this file getting below error, just FYI 
![image](https://github.com/mongodb/terraform-provider-mongodbatlas/assets/47370943/309008f9-4bf9-4b30-a14d-71928bf36ec3)

Reply:
Found the issue, fixing small indentation in that first yaml header section, thanks again for pointing out.